### DISCUSSION TITLE
Spec Profiles?

### DISCUSSION THREAD

**Discussion by NikolayPet** [SFX-WoW/Masque#29] [Questions] [answered]
Hello,
I use elvui (no addon skins) and weakauras. I use speck profiles for elvui and for masque. On my ele shaman I have sqaure borders coloured red. On my resto shaman I have circular borders coloured green. 

When I log in to elemental and respeck to resto masque does not skin the icons. When I type /masque in chat my borders change as soon as config window opens.

How do I fix that?

I skin the same frames - actionbars, auras and some weakaura icons.

**Comment by StormFX** [comment, marked as answer]
Masque itself doesn't support spec-level profiles. You need to have [LibDualSpec-1.0](https://www.wowace.com/projects/libdualspec-1-0) installed. If it is installed and it's not working, then it's an issue with the LibDualSpec. Note that it hasn't been updated in nearly two years, so it could very well be the issue.

**Comment by NikolayPet** [comment]
Oh, thanknyou for the reply. No  i do not habbe LibDualSpeck installed. Just elvui and weakauras. But since masque does not support spwck profiles I will just make some weakauras adjustments.
Anyways this topic is closed. You have been really helpfull.

**Comment by StormFX** [comment]
The next version will include LibDualSpec-1.0.
